analyzer/src/funTest/kotlin/managers/DotNetTest.kt

```diff
@@ -44,7 +46,10 @@ class DotNetTest : StringSpec() {
             val mapper = DotNetPackageReferenceMapper()
             val result = mapper.mapPackageReferences(packageFile)
 
-            result.size shouldBe 2
+            result should containExactly(mapOf(
```

Comment:
In the one other location with similar code

https://github.com/oss-review-toolkit/ort/blob/bd88d4bba90c3ed444555fe651954f15d58a3a8b/utils/src/test/kotlin/DeclaredLicenseProcessorTest.kt#L64

we use the `shouldContainExactly` single-word variant. Have we decided yet on which to align? Would you mind making an alignment commit and also remove the superfluous space before the parenthesis in the above code?

Reply:
For collections there is a vararg variant of `should containExactly` but not of `shouldContainExactly`, that's why I would prefer to align on the former. For maps there is no vararg variant, but we could make our own if we want. `should containExactly` is also dominant in the code base and better aligned with most other matchers we use. I'll make the alignment commit.